### PR TITLE
styling for event details page

### PR DESCRIPTION
**Pull request by ekong2**

### CODE REVIEW

File: client/public/assets/css/app.css [hunk 1]
```diff
@@ -133,4 +134,47 @@ body {
 
 #uploaded-picture {
 
-}
\ No newline at end of file
+}
+
+.event-detail-name *{
+  font-family: signpainter !important;
+  font-size: 300%;
```
Comment (kevhuang) [inline client/public/assets/css/app.css:141]: This is affecting the event create page.

**Comment by kevhuang** [comment]
On the event detail page, are the list of host and participants supposed to be to the right of the event details, or below the event details section? I'm seeing them as below the event details section.

**Comment by kevhuang** [comment]
:+1:

**Comment by dmsakamoto** [comment]
:+1:

**Pull request merged**